WeakAuras/WeakAuras.lua

```diff
@@ -1211,7 +1211,15 @@ function WeakAuras.CountWagoUpdates()
       end
       if slug and version then
         local wago = WeakAurasCompanion.slugs[slug]
-        if wago and wago.wagoVersion and tonumber(wago.wagoVersion) > tonumber(version) then
+        if wago and wago.wagoVersion
+        and tonumber(wago.wagoVersion) > tonumber(version)
```

Comment:
Isn't this effectively tonumber(wago.wagoVersion) > (aura.skipWagoUpdate and tonumber(aura.skipWagoUpdate) or tonumber(version))